README.md

```diff
@@ -1,40 +1,386 @@
-[![CloudNativePG](./logo/cloudnativepg.png)](https://cloudnative-pg.io/)
+# CloudNativePG Chaos Testing with Jepsen
 
-# CloudNativePG Chaos Testing
+![CloudNativePG Logo](logo/cloudnativepg.png)
 
-**Chaos Testing** is a project to strengthen the resilience, fault-tolerance,
-and robustness of **CloudNativePG** through controlled experiments and failure
-injection.
+Production-ready Jepsen and Litmus chaos automation for CloudNativePG (CNPG) clusters.
 
-This repository is part of the [LFX Mentorship (2025/3)](https://mentorship.lfx.linuxfoundation.org/project/0858ce07-0c90-47fa-a1a0-95c6762f00ff),
-with **Yash Agarwal** as the mentee. Its goal is to define, design, and
-implement chaos tests for CloudNativePG to uncover weaknesses under adverse
-conditions and ensure PostgreSQL clusters behave as expected under failure.
+---
+
+## 🚀 Quick Start
+
+**Want to run chaos testing immediately?** Follow these streamlined steps:
+
+1. **Setup cluster** → Bootstrap CNPG Playground (section 1)
+2. **Install CNPG** → Deploy operator + sample cluster (section 2)
+3. **Install Litmus** → Install operator, experiments, and RBAC (sections 3, 3.5, 3.6)
+4. **Smoke-test chaos** → Run the quick pod-delete check without monitoring (section 4)
+5. **Add monitoring** → Install Prometheus for probe validation (section 5; required before section 6 with probes enabled)
+6. **Run Jepsen** → Full consistency testing layered on chaos (section 6)
+
+**First time users:** Use section 4 as a smoke test without Prometheus, then return to section 5 to install monitoring before running the Jepsen workflow in section 6.
+
+**Troubleshooting?** Jump to the troubleshooting section for common issues and solutions.
+
+---
+
+## ✅ Prerequisites
+
+- Linux/macOS shell with `bash`, `git`, `curl`, `jq`, and internet access.
+- Container + Kubernetes tooling: Docker **or** Podman, the [Kind CLI](https://kind.sigs.k8s.io/) tool, `kubectl`, `helm`, the [`kubectl cnpg` plugin](https://cloudnative-pg.io/documentation/current/kubectl-plugin/) binary, and the [`cmctl` utility](https://cert-manager.io/docs/reference/cmctl/) for cert-manager.
+- Install the CNPG plugin if it is not already on your `PATH`:
+  ```bash
+  curl -sSL https://get.cnpg.io/install | sudo bash
+  kubectl cnpg version
+  ```
+  > If the installer endpoint is unreachable, download the **latest** release directly (replace `v1.27.1` with the newest tag at <https://github.com/cloudnative-pg/cloudnative-pg/releases>):
+  >
+  > ```bash
+  > VERSION="v1.27.1"
+  > curl -L "https://github.com/cloudnative-pg/cloudnative-pg/releases/download/${VERSION}/kubectl-cnpg_${VERSION}_linux_amd64.tar.gz" -o /tmp/kubectl-cnpg.tar.gz
```

Comment:
This command line doesn't work is not v1.27.1 for the version and amd64 isn't in the binary to download